viewer.py

```diff
@@ -606,12 +716,15 @@ def parseOntology(root, parent=-1):
     return ont
 
 
-def writeFile(data, file, **kwds):
```

Comment:
Since I updated the function and not passing any keywords to it, I decided to remove it. I could put it back(?)